src/components/layout/UserMenu.tsx

```
@@ -0,0 +1,139 @@
+import { useState, useRef, useEffect } from 'react';
+import { LogOut, User, Settings, HelpCircle, Moon, Sun, ChevronDown } from 'lucide-react';
```

ESLint

[error] 'ChevronDown' is defined but never used.

(@typescript-eslint/no-unused-vars)